packages/main/src/TableRow.js

```diff
@@ -31,6 +33,55 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.TableRow.prototype */ {
+		/**
+		 * Defines the mode of the row (None, SingleSelect, MultiSelect).
+		 * @type {TableMode}
+		 * @defaultvalue "None"
+		 * @private
+		 */
+		mode: {
+			type: TableMode,
+			defaultValue: TableMode.None,
+		},
+		/**
+		 * Defines the visual indication and behavior of the table row.
+		 * Available options are <code>Inactive</code> (by default) and <code>Active</code>.
+		 * <br><br>
+		 * <b>Note:</b> When set to <code>Active</code>, the item will provide visual response upon press,
+		 * while with type <code>Inactive</code> - will not.
+		 *
+		 * @type {TableRowType}
```

Comment:
add since especially here, but it won't hurt to add since for the private ones.

For since use 1.0.0-rc.14 (rc.13 has been released)